### DISCUSSION TITLE
support for `react-redux-firestore`

### DISCUSSION THREAD

**Discussion by PaulWozny** [invertase/react-native-firebase#6916] [Q&A] [answered]
Hi everyone

Today I found package that bridges redux and firestore: https://github.com/prescottprue/react-redux-firebase
Based on docs here https://react-redux-firebase.com/docs/integrations/react-native.html it seems that this package had support from `react-native-firebase`, at least 2 years ago:  https://github.com/prescottprue/react-redux-firebase/tree/master/examples/complete/react-native-firebase 

Does anyone use it?

Regards

**Comment by mikehardy** [comment, marked as answer]
I don't see why it wouldn't work? It should work, the API here is roughly the same as it was 2 years ago (the old style firebase-sdk v8 "chained" syntax) though we are headed towards firebase-js-sdk v9+ "modular" syntax.

We did switch to a monorepo here around the time they built their example (that was the big break between our "v5" version and v6+ here), so the imports will be different - `@react-native-firebase/app`, `@react-native-firebase/firestore` etc vs just monolithic `react-native-firebase` as an import.

I have no specific experience with this library though, that's just a general statement on how react-native-firebase has evolved (or not, as the case may be - with regard to general syntax style) over the last couple of years.

**Reply by PaulWozny** [reply]
Thanks for response! I'll give it a try.
